examples/vision_example.py

```diff
@@ -30,3 +31,4 @@
 )
 
 print(response.choices[0].message.content)
+print(response.choices[0].message.content)
```

Comment:
```suggestion
```

Reply:
Some issue with my local linter, i will fix that!